### ISSUE TITLE
Recipes use tag `c:yada_yada` instead of `forge:yada_yada` on forge

### ISSUE THREAD

**Issue by aaronhowser1** [SamsTheNerd/HexGloop#36]
![image](https://github.com/SamsTheNerd/HexGloop/assets/15698722/4acf95e7-9509-4c45-98de-9e54a01d196e)

**Comment by SamsTheNerd** [comment]
hm, looks like the recipe loads/works fine and just alternates between the gold nugget and the empty tag showing, is that the same behavior for you? 

currently I don't have my recipes split across platforms. I've been wanting to look into proper data gen in a future update, which would probably make this easier to fix.

**Comment by aaronhowser1** [comment]
Yes, but that tag doesn't exist in forge. In forge, it's like `forge:nuggets/gold` or something like that, which is what I meant the problem to be

**Comment by SamsTheNerd** [comment]
oh, yeah, it should have both? 
https://github.com/SamsTheNerd/HexGloop/blob/main/common/src/main/resources/data/hexgloop/recipes/casters_coin.json

The `c:tag` style is fabric's. idk why patchi only seems to show empty tags, or really why it shows them at all.

I'll run through the recipes again real quick before tonight's next release and make sure everything has a proper forge tag though.

**Comment by aaronhowser1** [comment]
It shows both vanilla gold ingots as well as the `c` one, my point was that it probably shouldn't use the `c` one as that doesn't exist in forge

**Comment by SamsTheNerd** [comment]
yeah probably not lol. I'd have to keep two separate json files for each recipe with a tag though, which would be a lot easier with datagen, so probably something i'll get around to whenever i switch over to that.
